external/anomaly/configs/draem/configuration.py

```diff
@@ -1,6 +1,4 @@
-"""
```

Comment:
Maybe we could move these files to `external/configs/base/draem`. That way the `configs` directory would be organized in a more consistent task -> algo structure.

Reply:
I tried that earlier but since all the model configurations depend on `base/configuration.py` I got error due to cyclic dependencies. Ideally I feel configuration.py and templates should be in a separate directory.

Reply:
This structure works fine when I try it:
![Screenshot from 2022-08-31 14-08-03](https://user-images.githubusercontent.com/26067904/187674874-52b3c203-462c-49cb-ad8f-8770da92acb5.png)

Reply:
I have the following in `configs/base/__init__.py`:
```
from .padim import PadimAnomalyBaseConfig
from .stfpm import STFPMAnomalyBaseConfig
from .draem import DraemAnomalyBaseConfig

__all__ = ["PadimAnomalyBaseConfig", "STFPMAnomalyBaseConfig", "DraemAnomalyBaseConfig"]
```
We don't need to import `BaseAnomalyConfig` in the `__init__.py`, because it's not used outside the `configs.base` module.

Reply:
Good point